### ISSUE TITLE
[3.5.0] Download of the geoip2 ISP/ASN database fails

### ISSUE THREAD

**Issue by tuxmaster** [matomo-org/matomo#12871]
The download self, will work, but the database will not detected:
Uses database: http://geolite.maxmind.com/download/geoip/database/GeoLite2-ASN.tar.gz
Error:

> No valid GeoIP database could be found in tar archive '/var/www/piwik/misc/GeoIP2-ISP.mmdb.tar.gz'!f

tar tzf GeoIP2-ISP.mmdb.tar.gz 
GeoLite2-ASN_20180508/
GeoLite2-ASN_20180508/COPYRIGHT.txt
GeoLite2-ASN_20180508/LICENSE.txt
GeoLite2-ASN_20180508/GeoLite2-ASN.mmdb

**Comment by sgiehl** [comment]
The ASN database is not yet supported. I will have a look at that for next release.

**Comment by tuxmaster** [comment]
Thanks for the sort answer.
Because I don't have find in ISP database at:
https://dev.maxmind.com/geoip/geoip2/geolite2/
Only an ASN database.

**Comment by sgiehl** [comment]
The ISP database is a paid one. It's not provided for free by maxmind

**Comment by fdellwing** [comment]
Yes please look into that @sgiehl the ISP Info can be found in the ASN database and was working in GeoIP Legacy. We need this info and cannot upgrade to GeoIP2 beforehand.
